hdf5.spec

```diff
@@ -283,9 +298,13 @@ do
   mkdir $mpi
   pushd $mpi
 %if (0%{?suse_version} >= 1500)
-  module load gnu-mpich
+  module load gnu-$mpi
 %else
-  module load mpi/$mpi-%{_arch}
+  if [ "$mpi" = "openmpi" ]; then
+    module load mpi/${mpi}3-%{_arch}
```

Comment:
I'm on the fence about this.  EL7 does have both an openmpi and openmpi3 package which are two different major versions of openmpi.  While I can't see us ever wanting to have openmpi (as opposed to openmpi3) support, overloading openmpi to really mean openmpi3 here gives me pause.

I noticed you did use openmpi3 as the variant name in a previous iteration of this patch.  Can I ask why you backed away from that?

Reply:
The changes from openmpi3 to openmpi was based on feedback from a previous review.   The change was requested due to  how mpich was referenced (mpich and mpich2).      I think that the name  should have openmpi3*.rpm (version 3.x) and if we decide we need it, openmpi (for version 1.x) in the package name.    I will update the package names.

Reply:
mpi_list will now have openmpi and openmpi3 and the above conditional is no longer needed.